pymc3/distributions/dist_math.py

```diff
@@ -73,16 +74,12 @@ def bound(logp, *conditions, **kwargs):
     """
 
     # If called inside a model context, see if bounds check is disabled
-    try:
+    if model!=None:
         from pymc3.model import modelcontext
 
-        model = modelcontext(kwargs.get("model"))
+        model = modelcontext(model)
```

Comment:
pylint will complain about the `!= None` comparison. Always use `is None` or `is not None` instead of `==`/`!=`.

After already establishing that there is a model, the `modelcontext` line can be removed. See [`def modelcontext`](https://github.com/pymc-devs/pymc3/blob/main/pymc3/model.py#L252-L264).

But what should happen when this function is called inside a modelcontext without passing the model object?
By doing
```
try:
    model = modelcontext(model)
except TypeError:
    # no model found
```
you could make that a little more explicit.

Reply:
Thanks @michaelosthege. I added these changes into the latest commit.